### PR TITLE
feat: rename my-account library to organization; rename organization feature to administration

### LINKED ISSUE #9340: Rename my-account library
We've started with the my-account library that would host various pieces that are available in the my account navigation. But we've realized lately that my-account isn't really a domain, it's more links to existing domains, such as user(profile), organization, orders, etc. 

Therefore, we like to change the library name and top level structure a bit.

```
- organization
   - administration
   - order-approval
```

We rename the `my-account` lib to `organization`.
We rename the `organization` entry to `administration`
We move the order-approval up, as it's not an administrative task

### PR DESCRIPTION
**Pull request by znikola**
closes #9340

### CODE REVIEW

File: core-libs/setup/test.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
-import '@angular/localize/init';
 import { getTestBed } from '@angular/core/testing';
+import '@angular/localize/init';
 import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone';
```
Comment (dunqan) [inline core-libs/setup/test.ts]: I don't think we have to change it?

File: core-libs/setup/.release-it.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
           "path": [
             "peerDependencies.@spartacus/core",
             "peerDependencies.@spartacus/storefront",
-            "peerDependencies.@spartacus/my-account"
+            "peerDependencies.@spartacus/organization"
```
Comment (marlass) [inline core-libs/setup/.release-it.json]: This line can be dropped. We forgot about it with lazy loading ticket. Setup lin no longer have dependency on organization

File: core-libs/setup/package.json [hunk 3]
```diff
@@ -21,6 +21,6 @@
     "rxjs": "^6.6.0",
     "@spartacus/core": "3.0.0-next.4",
     "@spartacus/storefront": "3.0.0-next.4",
-    "@spartacus/my-account": "3.0.0-next.4"
+    "@spartacus/organization": "3.0.0-next.4"
```
Comment (marlass) [inline core-libs/setup/package.json]: This dep can be removed

File: feature-libs/organization/administration/organization.module.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import { OrganizationOccModule } from '@spartacus/organization/administration/occ';
+
+@NgModule({
+  imports: [
+    OrganizationCoreModule.forRoot(),
+    OrganizationOccModule,
+    OrganizationComponentsModule,
+  ],
+})
+export class OrganizationModule {}
```
Comment (Platonn) [inline feature-libs/organization/administration/organization.module.ts:13]: Should this folder be still called `OrganizationModule`? Maybe `AdministrationModule` or `OrganizationAdministrationModule`?

It represents the entry point `@spartacus/organization/administration`

(same for child modules ...Core, Occ,Components)

File: feature-libs/product/test.ts [hunk 5]
```diff
@@ -6,8 +6,8 @@ import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone-testing';
```
Comment (Platonn) [inline feature-libs/product/test.ts]: Why the different lib now uses zone.js from organization library?

I've seen similar changes in:
```
integration-libs/cdc/src/test.ts
projects/storefrontapp/src/polyfills.ts
.... (and many more)
```

File: projects/storefrontapp/src/environments/b2b/b2b.feature.ts [hunk 6]
```diff
@@ -18,7 +18,7 @@ export const b2bFeature: FeatureEnvironment = {
       featureModules: {
         organization: {
```
Comment (Platonn) [inline projects/storefrontapp/src/environments/b2b/b2b.feature.ts:19]: Should we now call the cms feature `organization`? Maybe `organizationAdministration`?

File: feature-libs/organization/package.json [hunk 7]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@spartacus/my-account",
+  "name": "@spartacus/organization",
   "version": "3.0.0-next.4",
   "description": "My Account library for Spartacus",
```
Comment (marlass) [inline feature-libs/organization/package.json]:
```suggestion
  "description": "Organization library for Spartacus",
```

File: feature-libs/organization/administration/organization.module.ts (hunk 4 above)
Comment (znikola) [reply feature-libs/organization/administration/organization.module.ts:13]: We agreed offline to do it in the next iteration.

File: projects/storefrontapp/src/environments/b2b/b2b.feature.ts (hunk 6 above)
Comment (znikola) [reply projects/storefrontapp/src/environments/b2b/b2b.feature.ts:19]: We agreed offline to do it in the next iteration.

**Review by marlass — COMMENTED** [review]
Does styles work in storefrontapp?
There is still `my-account` reference there

File: feature-libs/product/test.ts [hunk 8]
```diff
@@ -6,8 +6,8 @@ import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone';
```
Comment (dunqan) [inline feature-libs/product/test.ts]: The same case as with core-libs

File: integration-libs/cdc/src/test.ts [hunk 9]
```diff
@@ -6,8 +6,8 @@ import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone';
```
Comment (dunqan) [inline integration-libs/cdc/src/test.ts]: another one ;-)

**Comment by Platonn** [comment]
In this file we should change to `@import './administration/index'`
https://github.com/SAP/spartacus/blob/feature/GH-9340/feature-libs/organization/_index.scss#L1

**Review by dunqan — APPROVED** [review]

**Review by Platonn — APPROVED** [review]

**Review by marlass — APPROVED** [review]

**Pull request merged**